cookbooks/aws-parallelcluster-platform/recipes/install/disable_services.rb

```diff
@@ -27,3 +27,13 @@
 service 'log4j-cve-2021-44228-hotpatch' do
   action %i(disable stop mask)
 end unless on_docker?
+
+# Disable services if node['cluster']['disable_services'] is provided
+if node['cluster']['disable_services']
```

Comment:
cookbook attributes are strings ('true', 'false'), so we should check `node['cluster']['disable_services'] == 'true'`.

See other cases https://github.com/aws/aws-parallelcluster-cookbook/blob/0079b13eafb897eb3294609efa6f53be5be21e02/cookbooks/aws-parallelcluster-slurm/templates/default/slurm/slurm.conf.erb#L23

Reply:
I am not keeping this attribute for boolean values. I am planning on using it like a string which contains the service names I want to disable

Reply:
They can also be expressed as boolean, so your change is fine.

See https://github.com/aws/aws-parallelcluster-cookbook/blob/1e41e652ea70b309338b6baf91242eb53c1675e5/cookbooks/aws-parallelcluster-entrypoints/attributes/entrypoints.rb#L20